pkg/adapterManager/manager.go

```diff
@@ -240,7 +240,7 @@ func (m *Manager) loadConfigs(attrs attribute.Bag, ks config.KindSet, isPreproce
 		return nil, fmt.Errorf("unable to resolve config: %v", err)
 	}
 	if glog.V(2) {
-		glog.Infof("Resolved %d configs: %v ", len(configs), configs)
+		glog.Infof("Resolved %d configs", len(configs))
```

Comment:
It's useful in that it tells us how many adapters we're dispatching to. The more detailed logs in `runtime.go` are at a higher verbosity, so won't be printed during normal operation, while this will.